ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/hosts.ini

```diff
@@ -107,6 +107,7 @@ route-maps
 router-bfd
 router-bgp-base
 router-bgp-evpn
+router-bgp-evpn-route-server
```

Comment:
could we do without a new device in molecule to just test "one knob" ? (I know adding it to the existing `router-bgp-evpn` might feel wrong but we really just need to check we generate the knob)

Reply:
Done although it is incompatible with the existing evpn config of "host" router-bgp-evpn